### PR TITLE
my dashboard: show record status in uploads list

### PR DESCRIPTION
**Pull request by NRodriguezcuellar**
closes: https://github.com/inveniosoftware/invenio-app-rdm/issues/1478

Screenshot from 2022-04-29 13-23-40

### CODE REVIEW

File: invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/components/uploads.js [hunk 1]
```diff
@@ -34,6 +34,15 @@ import {
 import { axiosWithconfig, SearchItemCreators } from "../../utils";
 import { DashboardResultView, DashboardSearchLayoutHOC } from "./base";
 
+const statuses = {
+  in_review: { color: "yellow", title: i18next.t("In review") },
```
Comment (ntarocco) [inline invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/components/uploads.js:38]: given that such colors are not used in a `grid`, are they matching the upload form?

**Review by kpsherva — COMMENTED** [review]
the label placement is quite different than in the mockup, and creates misalignment with the text below
in the mockup the status label was quite highlighted by separating its own area in the item
Screenshot 2022-04-28 at 18 02 49

File: invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/components/uploads.js (hunk 1 above)
Comment (Pineirin) [reply invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/components/uploads.js:38]: The colors roughly match the colors used in the upload form: https://github.com/inveniosoftware/react-invenio-deposit/blob/93d6a0d2ae057f4031b50e614aef33d4acb2a1ac/src/lib/components/DepositStatus/DepositStatusBox.js

**Comment by kpsherva** [comment]
we have now two labels with the same colour (red on `Declined` and `Metadata only`). Would it be an option to change `metadata-only` colour? Red usually indicates negative correlation in the interface, like "Decline" "delete" "remove" while metadata only is more an information. @ntarocco @lnielsen @jennur

File: invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/components/uploads.js (hunk 1 above)
Comment (kpsherva) [reply invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/components/uploads.js:38]: on another topic: those colors are not easily overridable or extendable in the current implementation, they are hardcoded

**Comment by jennur** [comment]
I agree the color of `metadata-only` could be a different color. Another suggestion is to maybe move the status label to the other side of the container to separate it from the metadata-labels.
Screenshot 2022-05-02 at 10 16 35

If we also remove the delete-button:
Screenshot 2022-05-02 at 10 26 30

**Comment by kpsherva** [comment]
@jennur I agree to remove the delete button since it is potentially dangerous action (does it even have a modal to confirm?)

I think however the status of the community submission status should stay on the left side as initially indicated in the mockups, since it is quite important information

File: invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/globals/site.variables [hunk 2]
```diff
@@ -24,10 +24,10 @@
 @signupColor: @green;
 @searchButtonColor: @secondaryColor;
 
-@accessRightOpen: @green;
-@accessRightRestricted: @red;
-@accessRightEmbargoed: @yellow;
-@accessRightMetadataOnly: @red;
+@accessRightOpen: @blue;
```
Comment (kpsherva) [inline invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/globals/site.variables:27]: please revert this, for the reasons discussed on the teams chat

**Comment by lhenze** [comment]
Hi, I suggest the text treatment of the labels (content type, metadata only, data) should also factor in whether they are actionable or not ...  there is a ticket somewhere for allowing some of them to be clickable and act like facets.  It makes sense to me that "Book" and "Metadata Only" act that way but that date doesn't.

**Pull request merged**